### ISSUE TITLE
The program doesn't detect song i'm playing

### ISSUE THREAD

**Issue by Cyber-Repocord** [kkevinm/iTunes-Discord-RP#47]
The program worked well at first but then it just stopped working, I tried everything to fix it, but i couldn't find a way. I don't know if it's a bug or if I messed something up, but it doesn't work (It says "Currently there's no song playing.", even though I'm playing a song, or sometimes it doesn't show anything but just blank field where the song name and author should be) (I am on Windows and it doesn't work on .jar and on .exe)

**Comment by jkelol111** [comment]
I'm on macOS 11 Big Sur with Music app version 1.1.1.3 having this same issue.

**Comment by vleeuwenmenno** [comment]
Yeah, I can confirm on Big Sur 11.0.1 it doesn't work (Apple Music 1.1.1.3)

**Comment by vleeuwenmenno** [comment]
Yeah, I can confirm on Big Sur 11.0.1 it doesn't work (APple Music 1.1.1.3)

**Comment by AshyAspen** [comment]
Can confirm here as well on a MacBook Air (Retina, 13-inch, 2019) running macOS Big Sur 11.0.1 (20B29) and Apple Music 1.1.1.3 - not working.

**Comment by jkelol111** [comment]
I upgraded to Big Sur 11.1 Public Beta, Music 1.1.2.25 and this is still happening.

**Comment by pounty** [comment]
This is certainly a bug introduced by Apple with Big Sur. The AppleScript can't get the current track information. Even the current track information using Automator doesn't work anymore. We just can wait for a fix by Apple...

**Comment by gtranche** [comment]
I have the same problem too, it seems the permission demand to access Music App is not the good one.
After I accept the permission, i can't see it it the Sercurity parameters.

**Comment by jkelol111** [comment]
@pounty is correct, I have created my own Python script using the `applescript` library  to do rich presence too, and it only works for songs you have saved in the library, for songs played elsewhere on Apple Music, it will fail to retrieve the song information.

The error I got was:
```
execution error: Can’t make name of «class pTrk» of application "Music" into type string. (-1700)
```

**Comment by pounty** [comment]
Can be closed now. Works again with macOS 11.1 👍

**Comment by jkelol111** [comment]
Doesn't work for me on 11.1, Music 1.1.2.28 unfortunately. Same behavior as before, if I'm playing a song in my library, it works, outside the library, it doesn't.

**Comment by kiwi2021** [comment]
I second this, the app is only working for music that has been downloaded, and anything being streamed is not shown on the app or discord. Hopefully this can be resolved!

**Comment by triscuitcircuit** [comment]
App only works with music thats downloaded throught the iTunes library for me as well

**Comment by sunsetsonwheels** [comment]
Seems to be fixed with 11.3, Music 1.1.4.94. Working with songs outside of library.
